renku/core/management/repository.py

```diff
@@ -407,6 +436,93 @@ def init_repository(self, force=False):
         # verify if author information is available
         Person.from_git(self.repo)
 
+    @property
+    def path_activity_cache(self):
+        """Cache of all activities and their generated paths."""
+        # TODO: this is there for performance reasons. Remove once graph
```

Comment:
Can we create a ticket for this and put reference to the comment?

Reply:
https://github.com/SwissDataScienceCenter/renku-python/issues/966#issuecomment-618906437 added here